src/pages/Pairs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext } from 'react'
```

Comment:
Wait, is this component duplicated from above?

Reply:
https://github.com/levelkdev/dxswap-dapp/pull/170/commits/2441099428f996a484f76aa5da4923ad90c09e3d